tests/test_app/TestApp/Template/Email/html/html.ctp

```diff
@@ -1,5 +1,5 @@
 <?php
-use Cake\Network\Email\Email;
+use Cake\Mailer\Email;
```

Comment:
Hmm - I guess with find and replace in entire project for namespace, there was a reference to `Cake\Network\Email\Email`.